### DISCUSSION TITLE
Exult 1.8 to 1.9 Config File no longer works?

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#350] [General]
Edit: I was wrong, I had a system message blocking the file copy.  :(

**Comment by DominusExult** [comment]
that's odd.
I'd say make a backup of the old cfg, and then delete one game section after the other to see where the culprit lies.
keeping it one <> tag per line would certainly help I guess.

**Reply by KnightCaptainU7** [reply]
User error, I didn't copy the folders completely.
